apps/web/pages/booking/[uid].tsx

```diff
@@ -176,13 +68,14 @@ const stringToBoolean = z
 
 const querySchema = z.object({
   uid: z.string(),
-  allRemainingBookings: stringToBoolean,
+  email: z.string().optional(),
+  eventTypeSlug: z.string().optional(),
```

Comment:
Add missing eventTypeSlug that wasn't needed at client but was needed at server. It comes from `schema` variable that has been removed.